src/constants/index.js

```diff
@@ -1034,6 +838,7 @@ export const projects = [
   },
 ];
 
+// You can add informative blog posts that you might want to display on your portfolio here
```

Comment:
```suggestion
// Add links to blogs here
```

Keep the tense constant while writing any documentation - somewhere its 'you can add', somewhere contain, somwhere else contains.

Subtle difference but neat.